### PR TITLE
change Report a Bug link and text in website FAQ

### PR DESCRIPTION
**Pull request by Alys**
Changes the text and link in the website's FAQ for the "How do I report a bug?" question. The link points to the new guild, and the text says, "To report a bug, go to [Help > Report a Bug](https://habitica.com/#/options/groups/guilds/a29da26b-37de-4a71-b0c6-48e72a900dac) and read the points above the chat box. Don't worry, we'll get you fixed up soon!"

after

@lemoness, would you like any further wording changes?

This is the version that's currently live:

before

### CODE REVIEW

**Comment by SabreCat** [comment]
The FAQ is on a static page accessible without being logged in. Opening a Guild requires that you be logged in. That means it's possible for someone to have a bug to report, but be unable to reach the target of the link--such as if the bug is preventing them from logging in. Should we include an alternate bug support link for that situation?

**Comment by Alys** [comment]
Oh, good point! I've added a note about emailing us. It now says:

To report a bug, go to [Help > Report a Bug](https://habitica.com/#/options/groups/guilds/a29da26b-37de-4a71-b0c6-48e72a900dac) and read the points above the chat box. If you're unable to log in to Habitica, send your login details to admin@habitica.com. Don't worry, we'll get you fixed up soon! 
[Then the same Trello information as before.]

**Comment by crookedneighbor** [comment]
Login details sounds like sending in username and password to me.

**Comment by Alys** [comment]
Hmmm, yeah. We would typically want email address and/or username (both are best) or to be told that they use Facebook login, but that's a bit wordy. I guess just "If you're unable to log in to Habitica, email admin@habitica.com". I'll wait for more feedback before changing it again.

**Comment by crookedneighbor** [comment]
I think that's fine. The important thing is to email. If we need more info than they give, we can follow up.

**Comment by khipkin** [comment]
#7719 is semi-related to this, assuming the iOS and Android FAQ should be updated in similar ways once this is merged in.

**Comment by lemoness** [comment]
I think leaving it as "email admin@habitica" is definitely the way to go.
Other wording changes look fine!

**Comment by Alys** [comment]
I'll finish this off and fix merge conflicts after the other FAQ PR has been merged.

**Comment by Alys** [comment]
The other PR (https://github.com/HabitRPG/habitrpg/pull/7726) has merge conflicts already so I'm merging this one now.
